src/libs/PDCurses/sdl2_queue/pdckbd.cpp

```diff
@@ -373,16 +379,16 @@ static int _process_mouse_event(void)
 
         if (action == BUTTON_PRESSED && SP->mouse_wait)
         {
-            SDL_Event rel;
-
             napms(SP->mouse_wait);
 
-            if (SDL_PollEvent(&rel))
+            if (!pdc_event_queue.empty())
             {
-                if (rel.type == SDL_MOUSEBUTTONUP && rel.button.button == btn)
+                const auto rel = pdc_event_queue.front();
```

Comment:
Same.
Event is called `ev` above but `rel` here.  I can't tell from this diff what `rel` means (possible to still call it `ev`?)

Reply:
They called it `rev` here so I stuck with it